pkg/minishift/addon/addon_meta.go

```diff
@@ -147,6 +152,23 @@ func (meta *DefaultAddOnMeta) MinishiftVersion() string {
 	return anyMinishiftVersion
 }
 
+func (meta *DefaultAddOnMeta) Dependency() ([]string, error) {
+	if val, contains := meta.headers[dependsOn].(string); contains {
+		return minishiftStrings.SplitAndTrim(val, ",")
+	}
+	return []string{}, nil
+}
+
+func checkDependencySemantic(headers map[string]interface{}) bool {
+	// Comma seperated list of dependencies
+	if headers[dependsOn] != nil {
+		dependencies := headers[dependsOn].(string)
+		match, _ := regexp.MatchString("^(([a-zA-Z][-[a-zA-Z]*)([,][ ]?[a-zA-Z][-[a-zA-Z]*)*)$", dependencies)
```

Comment:
maybe add some clarification for the expectation?

Reply:
any error reporting when the format is incorrect?

Reply:
@gbraad No, it does not report what is wrong with the format, just that it is wrong. expected is a comma separated list as in `anyuid,xpaas` or `anyuid, xpaas` (i.e one space is accepted) but it should not end with a comma.